### PR TITLE
Fix Superset image and upgrade to 1.4.1

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

File: superset/Makefile [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+	$(BUILDER) build -t $(IMAGE_NAME) .
+
+push:
+	echo "Pushing $(IMAGE_NAME) image..."
+	$(BUILDER) push $(IMAGE_NAME)
 
-.PHONY: test
 test:
-	container-structure-test test -i $(IMAGE_NAME) -c test-config.yaml
+	container-structure-test test -i $(IMAGE_NAME) -c test-config.yaml
```
Comment (anishasthana) [inline superset/Makefile]: Missing new line

File: superset/README.md [hunk 2]
```diff
@@ -29,7 +20,7 @@ to test built application image:
 ### Running the application image
 Running the application image is as simple as invoking the docker run command:
 ```
-docker run -d -p 8080:8080 <your-desired-image-name>
+docker run -d -p 8088:8088 <your-desired-image-name>
 ```
 
 The application should now be accessible at  [http://localhost:8080](http://localhost:8080).
```
Comment (anishasthana) [inline superset/README.md]: The command above is specifying `port 8088` but in the doc we are saying `8080`

File: superset/README.md [hunk 3]
```diff
@@ -29,7 +20,7 @@ to test built application image:
 ### Running the application image
 Running the application image is as simple as invoking the docker run command:
 ```
-docker run -d -p 8080:8080 <your-desired-image-name>
+docker run -d -p 8088:8088 <your-desired-image-name>
```
Comment (anishasthana) [inline superset/README.md]: Why not `podman` :-)

File: superset/Makefile [hunk 4]
```diff
@@ -1,9 +1,17 @@
-IMAGE_NAME?=centos/python-38-centos7
+SUPERSET_VERSION=1.4.1-ubi
+IMAGE_NAME=quay.io/opendatahub/superset:${SUPERSET_VERSION}
+
+BUILDER:=$(shell command -v docker 2> /dev/null || command -v podman 2> /dev/null || echo "docker or podman not found")
```
Comment (anishasthana) [inline superset/Makefile]: Could we swap the other on this to check for podman first?

**Review by anishasthana — CHANGES_REQUESTED** [review]

**Comment by anishasthana** [comment]
The make build is failing for me

```
Errors during downloading metadata for repository 'rhel-8-for-x86_64-baseos-beta-rpms':
  - Status code: 403 for https://cdn.redhat.com/content/beta/rhel8/8/x86_64/baseos/os/repodata/repomd.xml (IP: 23.49.52.251)
Error: Failed to download metadata for repo 'rhel-8-for-x86_64-baseos-beta-rpms': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

What specific rights do we need?

**Comment by rimolive** [comment]
@anishasthana I'm not sure where/how you tried to build the image, but I got a fresh CoreOS VM and I could build the image with no issues.

I know that UBI usually relies on the repos that are enabled in the host, so that might be what happened to you.

**Comment by LaVLaS** [comment]
I did get a similar error with a different repo.  I think it may have been an issue to the repo mirror redirection.  

```
Errors during downloading metadata for repository 'rhel-8-for-x86_64-appstream-beta-rpms':
  - Status code: 403 for https://cdn.redhat.com/content/beta/rhel8/8/x86_64/appstream/os/repodata/repomd.xml (IP: 173.222.100.251)
Error: Failed to download metadata for repo 'rhel-8-for-x86_64-appstream-beta-rpms': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

I think the presence of subscription-manager on the host machine is causing issues with the reposync process

**Review by anishasthana — COMMENTED** [review]
I was able to successfully build on another machine. /lgtm

**Review by anishasthana — APPROVED** [review]

**Pull request merged**